pkg/node/smart.go

```diff
@@ -100,8 +107,14 @@ func (s *SmartService) GetDriveSmartInfo(ctx context.Context, params smart.GetDr
 
 	s.log.Debugf("Drivemgr response %v ", smartInfoResponse)
 
+	smartInfoRawJSON := smartInfoResponse.GetSmartInfo()
```

Comment:
do you see a possibility to extract duplicated code into the separate  method? this 3 lines smartInfoRawJSON := smartInfoResponse.GetSmartInfo()
	if !json.Valid([]byte(smartInfoRawJSON)) {
		s.log.Errorf("Unable to parse a JSON returned from drivemgr: %v", smartInfoRawJSON)
		can return true or false